targets/nuttx-stm32f4/nuttx/config_mpu

```diff
@@ -0,0 +1,1673 @@
+#
+# Automatically generated file; DO NOT EDIT.
```

Comment:
Done.
I'd say I am not sure starting file names with the dot '.' is a good idea, as they do not show when listing directories with ls (standard practice on unix/OSX).